src/test/java/cart/data/MemberJdbcRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package cart.data;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseBuilder;
+import org.springframework.jdbc.datasource.embedded.EmbeddedDatabaseType;
+
+import javax.sql.DataSource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+class MemberJdbcRepositoryTest {
```

Comment:
`@JdbcTest` 를 활용하면 Repository 빈을 을 주입받아 사용할 수 있습니다 :)
스프링에서 테스트를 쉽게할 수 있도록 제공하는 기능이죠 😄 

+) `@Sql` 애너테이션을 사용하면 sql 스크립트 실행이 가능합니다 :)

단 `@JdbcTest` 애너테이션을 사용하면 테스트 실행 전~후 를 트랜잭션 범위로 잡아서
각 테스트 메서드 실행 후 롤백을 실행합니다.

즉, 각 테스트 메서드에서 실행한 쿼리들은 롤백이 되어 테스트 독립적으로 실행이 가능해집니다.

sql 스크립트 대신 `@BeforeEach`에서 더미데이터를 넣어주는건 어떨까요?

```suggestion
@Sql("classpath:test-data.sql")
@JdbcTest(
        includeFilters = {
                @ComponentScan.Filter(type = FilterType.ANNOTATION, value = Repository.class)
        }
)
class MemberJdbcRepositoryTest {

    @Autowired
    private MemberRepository memberRepository;
```